scripts/README.md

```diff
@@ -71,3 +71,41 @@ EVC_IDS='d33539656d8b40,095e1d6f43c745' priority python3 scripts/002_unset_spf_a
 ```
 
 - After that, `kytosd` should be restarted just so `mef_eline` EVCs can get fully reloaded in memory with the expected primary and secondary constraints, this would be the safest route.
+
+### Change ``tag_type`` from integer to string type
+
+[`vlan_type_string.py`](./vlan_type_string.py) is a script to change every ``tag_type`` instance from integer to string type. These istances are found in evcs collection from MongoDB.
+
+```
+    VLAN = 1 to 'vlan'
+    VLAN_QINQ = 2 to 'vlan_qinq'
+    MPLS = 3 to 'mpls'
+```
+
+#### Pre-requisites
+
+- Make sure MongoDB replica set is up and running.
+- Export the following MongnoDB variables accordingly in case your running outside of a container
+
+```
+export MONGO_USERNAME=
+export MONGO_PASSWORD=
+export MONGO_DBNAME=napps
+export MONGO_HOST_SEEDS="mongo1:27017,mongo2:27018,mongo3:27099"
+```
+
+#### How to use
+
+The following `CMD` commands are available:
+
+```
+CMD=aggregate_int_vlan python3 scripts/vlan_type_string.py
+```
+`aggregate_int_vlan` command is to see which EVC needs to be changed whether they have an outdated value type for a TAG inside their `uni_a`, `uni_z`, `current_path`, `failover_path`, `backup_path` or `primary_path`.
+
+```
+CMD=update_database python3 scripts/vlan_type_string.py
```

Comment:
Aldo, if you could follow the same script naming pattern that mef_eline has been using with a 3-digit number in front of it like `003_vlan_type_string.py`, this was also to facilitate understanding out of the gate their order. 

In the future, we do need to make this better though accross all NApps, let's start a new issue to categorize better the folder and to which version they belong, just so it becomes easier to also link and mention in the release notes (this part specifically won't be a blocker for this PR).

Reply:
Changed name